recipes/ignition-tools/all/conanfile.py

```diff
@@ -1,21 +1,23 @@
 import os
 from conans import CMake, ConanFile, tools
 from conans.errors import ConanInvalidConfiguration
+import conan.tools.files
+import textwrap
 
 required_conan_version = ">=1.29.1"
```

Comment:
the new properties being used are higher

```suggestion
required_conan_version = ">=1.43.0"
```

Reply:
done